### ISSUE TITLE
remember command history between sessions

### ISSUE THREAD

**Issue by stax76** [rossy/mpv-repl#1]
Hello,

remember command history between sessions would be very useful I think. Awesome extension anyway!

**Comment by ghost** [comment]
I actually want that sometimes too.

**Comment by rossy** [comment]
Yeah, this would be useful. If I add this feature, I want to do it right though, and it could be complicated. If there are two instances of mpv, for example, I think I'd want them to both atomically append to the history file, and be able to read each other's commands. They also should probably trim the history file to a maximum number of lines.

Permanent command history should probably come with a search feature too, like Ctrl+R in readline.
